### ISSUE TITLE
Render comment markdown

### ISSUE THREAD

**Issue by OfficerHalf** [OfficerHalf/obsidian-trello#3]
This is easy to achieve, but problematic.
``` typescript
const textContainer = commentContainer.createDiv('trello-comment--text-container');
const text = textContainer.createEl('div', {
  cls: 'trello-comment--text'
});
MarkdownRenderer.renderMarkdown(comment.data.text, text, '', this);
```

This works great, but styling is all over the place. A few things that need to be handled:

- Element margins
- Text size
- Probably a "see more" / "see less" type function to show/hide entire comment text if its super large

**Comment by OfficerHalf** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [render comment markdown](https://trello.com/c/u0oKM79Z/13-render-comment-markdown)

**Comment by OfficerHalf** [comment]
Done in #38, released in v1.6.0.
